src/Chapter1.hs

```diff
@@ -641,7 +657,11 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+
+firstDigit :: Int -> Int
+firstDigit n
+  | (abs n) < 10 = abs n
+  | otherwise = firstDigit (div (abs n) 10)
```

Comment:
That is right 💯 

You can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼

Reply:
Thank for you the suggestion! I will make changes accordingly.